### ISSUE TITLE
Request for confirmation of a flow in the Wallet

### ISSUE THREAD

**Issue by swcurran** [bcgov/bc-wallet-mobile#1620]
I've been asked to check with the BC Wallet team to confirm the following.

If I'm a verifier and I request a presentation based on a revocable credential, but I explicitly DON'T request the revocation status (e.g., I don't include any revocation interval in the presentation request), does the BC Wallet collect revocation data from the ledger in preparing the presentation, and does it provide a non-revocation proof?

**ANSWER**: Confirmed that there is **no** non-revocation proof included in the presentation if the verifier does not ask for it, and that the wallet does not retrieve the RevReg when constructing the proof.

Yay!

**Comment by swcurran** [comment]
Working on it...almost done.

**Comment by swcurran** [comment]
Hmmm... a new problem.  When I send the Presentation Request without revocation, everything works. When I add revocation at the attribute or presentation request level, I get an error in the wallet about an invalid URL.  Here is an example, non-working request -- removing the "," and "non_revoked" item works fine.

I'm guessing it is the URL in the credential definition from Traction -- investigating. 

```
{
  "nonce": "1234567890",
  "name": "proof-request",
  "version": "1.0",
  "requested_attributes": {
    "studentInfo": {
      "names": [
        "given_names",
        "family_name"
      ],
      "restrictions": [
        {
          "schema_name": "student id"
        }
      ],
      "non_revoked": {
        "from": 1700849626,
        "to": 1700849626
      }
    }
  },
  "requested_predicates": {
    "not_expired": {
      "name": "expiry_dateint",
      "p_type": ">=",
      "p_value": 20230527,
      "restrictions": [
        {
          "schema_name": "student id"
        }
      ]
    }
  }
}

```

**Comment by swcurran** [comment]
Confirmed that there is no non-revocation proof included in the presentation if the verifier does not ask for it.  Yay!

Along the way, ran into an issue with revocation, where the revocation interval was "too soon".  Eventually it worked, but strange that it took so long.  I tried a time in the future and that didn't work.  I queried the time the RevReg was created and added a bit of time to that, but that didn't work.  Looks like it needed to be 10 minutes after?  Weird...and not good.  Not sure what to do about that.
